orc8r/gateway/c/common/service303/test/test_service303.cpp

```diff
@@ -11,21 +11,33 @@
  * limitations under the License.
  */
 
-#include <unistd.h>
-#include <pthread.h>
-#include <prometheus/registry.h>
+#include <assert.h>
+#include <grpcpp/create_channel.h>
+#include <grpcpp/impl/codegen/client_context.h>
+#include <grpcpp/impl/codegen/status.h>
+#include <grpcpp/security/credentials.h>
 #include <gtest/gtest.h>
-#include <map>
+#include <metrics.pb.h>
+#include <orc8r/protos/common.pb.h>
 #include <orc8r/protos/metricsd.pb.h>
+#include <orc8r/protos/service303.grpc.pb.h>
+#include <orc8r/protos/service303.pb.h>
+#include <unistd.h>
```

Comment:
I guess, `<unitstd.h>` is a system header and thus should be above together with `<assert.h>`.

Reply:
same problem here. I think all of them here are classified as system headers and then alphabetically sorted. Will see if I can mark protobuf files as not system headers (and look into how these generated files tended to get sorted in other projects)

Reply:
@ajahl So we may have to take a 2 step approach here with @electronjoe 's include sorting script. So dev workflow might be

1. Make changes
2. Run IWYU (dev_tools/apply-iwyu.sh) -- responsible for getting all the necessary includes, not sorting
3. Run sorting script -- sort and modify protobuf includes to be with " and not <

Reply:
Sounds good to me.

Reply:
`So we may have to take a 2 step approach` ... This is what I'm exploring now - specifically running IWYU and then my Go binary (which is in progress) to fixup the outputs of IWYU.  Still in progress so we'll see if I run into any blockers here.